IdentityCore/IdentityCore.xcodeproj/project.pbxproj

```diff
@@ -450,6 +450,14 @@
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->